beacon-chain/utils/clock.go

```diff
@@ -1,9 +1,13 @@
 package utils
 
 import (
+	"math"
 	"time"
 )
 
+// GenesisTime used by the protocol.
+var GenesisTime = time.Date(2018, 9, 0, 0, 0, 0, 0, time.UTC) // September 2018
```

Comment:
I think GenesisTime should be a constant under `config.go`
https://notes.ethereum.org/SCIg8AH5SA-O4C1G1LYZHQ?view#Constants
What do you think?

Reply:
yeah, that's probably the best place to have it if someone wants to find it